Dockerfile

```diff
@@ -1,192 +1,170 @@
+        ${USERNAME}
+
 # Ensure that the environment uses UTF-8 encoding by default
 ENV LANG en_US.UTF-8
-ENV ENV /etc/profile
-ENV PIP_NO_CACHE_DIR true
-# Stops Python default buffering to stdout, improving logging to the console.
-ENV PYTHONUNBUFFERED 1
-ENV APP_HOME /src/consumerfinance.gov
-
-WORKDIR ${APP_HOME}
-
-# Install and update common OS packages, pip, setuptools, wheel, and awscli
-RUN apk update --no-cache && apk upgrade --no-cache
-RUN pip install --upgrade pip setuptools wheel awscli
-
-# Add `$HOME/.local/bin` to PATH
-RUN echo 'export PATH=$HOME/.local/bin:$PATH' >> /etc/profile
 
-#######################################################################
-# Intermediate layer to build only prod deps
-FROM base as cfgov-python-builder
-
-# .build-deps are required to build and test the application (pip, tox,
-# matplotlib, numpy, etc.)
-# TODO: When we update from the python:3.8 image, see which of these we can
-# remove because binary packages might be available to us again.
-RUN apk add --no-cache --virtual .build-deps gcc gettext git libffi-dev musl-dev postgresql-dev musl-dev linux-headers g++ make freetype
+# Disable pip cache dir
+ENV PIP_NO_CACHE_DIR 1
 
-# Install python requirements
-COPY requirements requirements
-RUN mkdir /build && pip install --prefix=/build -r requirements/deployment.txt
+# Stops Python default buffering to stdout, improving logging to the console
+ENV PYTHONUNBUFFERED 1
 
-#######################################################################
-# cfgov-dev is used for local development, as well as a base for frontend.
-FROM cfgov-python-builder AS cfgov-dev
+# Set the APP_HOME, our working directory
+ENV APP_HOME /src/consumerfinance.gov
 
-ENV CFGOV_PATH ${APP_HOME}
-ENV CFGOV_CURRENT ${APP_HOME}
+# Add our top-level Python path to the PYTHONPATH
 ENV PYTHONPATH ${APP_HOME}/cfgov
 
-# Django Settings
-ENV DJANGO_SETTINGS_MODULE cfgov.settings.local
-ENV ALLOWED_HOSTS '["*"]'
-
-# .backend-deps and .frontend-deps are required to run the application
-RUN apk add --no-cache --virtual .backend-deps bash curl postgresql
-
-# Install python requirements
-COPY requirements requirements
-RUN cp -Rfp /build/* /usr/local && rm -Rf /build && pip install -r requirements/local.txt
+# Set the working directory
+WORKDIR ${APP_HOME}
 
+# Copy the application requirements, needed to build the Python environment
+# below
+# Note: by specifying specific files we enable this layer to be cached if
+# these files do not change.
+COPY requirements ./requirements
+
+# Build the Python environment
+RUN \
+    apk update --no-cache && apk upgrade --no-cache && \
+    apk add --no-cache --virtual .build-deps \
+        gcc \
+        gettext \
+        git \
+        libffi-dev \
+        musl-dev \
+        postgresql-dev \
+    && \
+    apk add --no-cache --virtual .backend-deps \
+        bash \
+        curl \
+        postgresql \
+    && \
+    pip install --upgrade pip setuptools wheel awscli && \
```

Comment:
What is `awscli` for? I see that it exists in the existing Dockerfile. Is it just a debugging tool?

Reply:
Hmm, I think I just kept this line from the old Dockerfile. I don't think awscli is used.

Reply:
Removed in [37ef717](https://github.com/cfpb/consumerfinance.gov/pull/8676/commits/37ef71714dec0de4c40724d89013a9ea9bc2b28b)